perlio.h

```diff
@@ -63,6 +63,16 @@ typedef PerlIOl *PerlIO;
 #define PerlIO PerlIO
 #define PERLIO_LAYERS 1
 
+/*
+=for apidoc_section $io
+=for apidoc Amu||PERLIO_FUNCS_DECL|PerlIO * func
+Declare C<func> to be a PerlIO function, that is, of type C<PerlIO_funcs>.
```

Comment:
It doesn't define a function, it defines a PerlIO vtable

Reply:
@khwilliamson, can you respond to @LeonT's question -- and also advise as to the status of this p.r.?